src/day8/re_frame/trace.cljs

```diff
@@ -215,43 +216,61 @@
                                           tag-str))]]
                                [:td.trace--meta.trace--details-icon
                                   {:on-click #(.log js/console tags)}]]))))))))
+
 (defn render-trace-panel []
   (let [filter-input               (r/atom "")
         filter-items               (r/atom (localstorage/get "filter-items" []))
         filter-type                (r/atom :contains)
         input-error                (r/atom false)
+        categories                 (r/atom #{:event :sub/run :sub/create})
```

Comment:
should I leave a comment saying that events and subscriptions are being shown as default?

Reply:
I think it's okay as it is!